packages/dx-react-common/docs/guides/localization.md

```diff
@@ -0,0 +1,50 @@
+# React Components - Localization
+
+This article describes how to localize UI texts in DevExtreme React Components.
+
+## The "Messages" Property
+
+Plugins that render UI texts have the `messages` property. It accepts an object that contains messages as key-value pairs in which keys are message IDs and values are the messages. For a list of message IDs that a plugin supports, see the Localization Messages section of the plugin's API Reference.
+
+To translate a message, find the plugin the renders it and specify the plugin's `messages` property with an object that contains your translations. Those message IDs that you omit from the object use default translations.
```

Comment:
"plugin the renders" do you mean 'that' or 'which'?

Reply:
```suggestion
To translate a message, find the plugin that renders it and specify the plugin's `messages` property with an object that contains your translations. Messages without IDs use default translations.
```

Reply:
> "plugin the renders" do you mean 'that' or 'which'?

You guessed correctly.

> Messages without IDs use default translations.

There cannot be a message without an ID. There can only be a message whose ID is not included in the object specified by the developer.

Reply:
```suggestion
To translate a message, find the plugin that renders it and specify the plugin's `messages` property with an object that contains your translations. Those message IDs that you omit from the object use default translations.
```